addons/project_todo/static/tests/mock_server/mock_models/project_task.js

```diff
@@ -0,0 +1,31 @@
+import { fields } from "@web/../tests/web_test_helpers";
+import { projectModels } from "@project/../tests/project_models";
+
+
+export class ProjectTask extends projectModels.ProjectTask {
+    _name = "project.task";
+
+    company_id = fields.Many2one({ string: "Company", relation: "res.company" });
+    tag_ids = fields.Many2many({ relation: "project.tags" });
+
+    _records = [
+        {
+            id: 1,
+            name: "Todo 1",
+            state: "01_in_progress",
+            tag_ids: [1],
+        },
+        {
+            id: 2,
+            name: "Todo 2",
+            state: "1_done",
+            tag_ids: [3],
+        },
+        {
+            id: 3,
+            name: "Todo 3",
+            state: "01_in_progress",
+            tag_ids: [3,2],
```

Comment:
```suggestion
            tag_ids: [3, 2],
```

it does not seem you use the linting tool made by framework js. 🤔 

to install it you have to launch the following file in your terminal

https://github.com/odoo/odoo/blob/fbfef09ec52965745ba35727f9d2b1fa1c62ff83/addons/web/tooling/enable.sh#L0-L1